### ISSUE TITLE
Need help: spell checker for mixed languages

### ISSUE THREAD

**Issue by Mottotime** [brrd/Abricotine#192]
Would you please add support for spell checking on mixed languages?
I tried to add extra locale in the preference file, but it does not work.
Thanks a lot.

**Comment by andreaspada** [comment]
Yes, please! It will be great! I found myself writing in as much in four languages in the same document. Quite an headache...
